trade-module/src/main/java/io/edpn/backend/trade/adapter/persistence/filter/MybatisPageFilter.java

```diff
@@ -0,0 +1,22 @@
+package io.edpn.backend.trade.adapter.persistence.filter;
+
+import io.edpn.backend.trade.application.dto.persistence.filter.PersistencePageFilter;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@EqualsAndHashCode
```

Comment:
As much as I hate writing boilerplate, this sort of ADD (not attention deficit disorder, Annotation Driven Development!) makes me cringe, too. IIRC, we have some limitations wrt. using records, but do they apply here?